core/transfer/src/main/java/org/eclipse/dataspaceconnector/transfer/core/CoreTransferExtension.java

```diff
@@ -100,7 +96,7 @@ public void initialize(ServiceExtensionContext context) {
 
         var vault = context.getService(Vault.class);
 
-        provisionManager = new ProvisionManagerImpl();
+        ProvisionManagerImpl provisionManager = new ProvisionManagerImpl();
```

Comment:
A bit of a nit: since the existing code uses var, can you stick to that so it is consistent w/in the class?